docs/content/dagster-plus/deployment/azure/aks-agent.mdx

```diff
@@ -0,0 +1,40 @@
+# Dagster+ on Azure - Quickstart Guide
+
+This guide will walk you through deploying a Dagster+ agent on an Azure Kubernetes Service (AKS) cluster.
+
+This guide is intended to be a quickstart, and you should always defer to organization-specific guidelines for creating and managing new infrastructure.
+
+We'll start from a brand new organization in Dagster+, and finish with a full hybrid deployment of Dagster+ using Azure infrastructure.
+
+## Prerequisites
+
+To complete the steps in this guide, you'll need:
+
+- The azure CLI installed on your machine. You can download it [here](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli).
+- `kubectl` installed on your machine. You can download it [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+- `helm` installed on your machine. You can download it [here](https://helm.sh/docs/intro/install/).
+- An existing AKS cluster. If you need to create a new AKS cluster, refer to the [Azure documentation](https://learn.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-portal?tabs=azure-cli).
+- A Dagster+ organization, with an agent token for that organization.
+
+## Step 1: Generate a Dagster+ agent token.
+
+<GenerateAgentToken />
+
+## Step 2: Log in to your AKS cluster.
+
+We'll use the azure CLI to log in to your AKS cluster. Run the following command and follow the prompts to log in.
+
+```bash
+az login
+az aks get-credentials --resource-group <your-resource-group> --name <your-aks-cluster>
+```
+
+We should now be able to verify our installation by running a command that tells us the current context of our kubectl installation. We'd expect it to output the name of the AKS cluster.
+
+```bash
+kubectl config current-context
+```
+
+## Step 3: Install the Dagster+ agent on the AKS cluster.
+
+Next, we'll install the agent helm chart. You should be able to follow the guide [here](/concepts/deployment/agents/kubernetes/configuring-running-kubernetes-agent) to install the agent on the AKS cluster.
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +41]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->nit: feels a bit dry. maybe something like. 

"Once you've completed this step, you should have a running Dagster+ agent ready to accept your code locations, which you will deploy in (link to CI/CD)" ? <!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/26186?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
nvm - already done upstack